src/auth/auth.controller.ts

```diff
@@ -36,6 +38,20 @@ export class AuthController {
     return ZaLaResponse.Ok(userSignIn, 'Successfully logged in', 201);
   }
 
+  @Post('/signout')
+  @ApiOperation({description: 'Sign out a user'})
+  async signOutUser(@Body('refreshToken') refresToken:string) {
```

Comment:
Nice work, just one problem.

There is no field to enter the refresh token, fix your line 43

Reply:
No field ??

I did test before I pushed... I used postman to test each endpoint before making this PR.

Reply:
So where did you enter the token? as a body or as param? You have body but no Dto, and the field you are extracting from the body `refreshToken` doesn't match with variable name `refresToken`. Yet you insist you tested this? hmmm.

Reply:
Was working on another branch...